main.js

```diff
@@ -0,0 +1,243 @@
+//set all question card display to none so it won't appear on the HTML document
+let allcard = document.querySelectorAll("article.card")
+for (let eleemnt of allcard){
+    eleemnt.style.display = "none"
+}
+
+
+////selects all the question cards and display to block when the "Get New Questions" button is clicked
+let gettheQUESTIONS = document.querySelector("form button")
+gettheQUESTIONS.addEventListener("click", ()=>{
+    let allcard = document.querySelectorAll("article.card")
+    for (let eleemnt of allcard){
+        eleemnt.style.display = "block"
+    }
+})
+
+
+document.querySelectorAll(`article.card p div`).display = "none"
+
+document.querySelectorAll(`article.card p.hidden`).display = "none"
+
+//get the Trivia Question aPI
+fetch("https://opentdb.com/api.php?amount=10&category=17")
+
+//takes the Trivia API and turns the JSON into JAvascript with the JSON()
+.then(response =>{
+ return response.json();
+
+})
+//will retrieve all objects initself initially until we log the reults object
+.then((resultsArray) =>{  
+var gettingQuestion = resultsArray.results
+let answersArray = []
+let questionsArray = []
+console.log(resultsArray)
```

Comment:
Remember to remove any console.logs that don't have to do with completing the lab when you submit